content/SCALE/SCALETutorials/Shares/SMB/AddSMBHomeShare.md

```diff
@@ -14,103 +14,159 @@ They are not recommended for new deployments.
+## Replacing SMB Home Shares
+TrueNAS does not recommend setting up home shares with the **Use as Home Share** option, found in the **Add SMB** and **Edit SMB** screen **Advanced Options** settings in the **Other Options** section.
+This option is for organizations still using the legacy home shares option to use a single SMB share to provide a personal directory to every user account.
 
-With home shares, each user is given a personal home directory when connecting to the share.
+Users wanting to create the equivalent of home shares should use the intructions in the [Adding Private SMB Datasets and Shares](#adding-private-smb-datasets-and-shares) section below for the recommended method for creating private shares and datasets for users with this requirement.
+
+The legacy home shares provides each user a personal home directory when connecting to the share.
 These home directories are not accessible by other users.
 You can use only one share as the home share, but you can create as many non-home shares as you need or want.
 
-<!-- Place holders for future tutorials/links below. Update with links to appropriate procedures as part of PD-1252 -->
 Other options for configuring individual user directories include:
 * Configure a single share on the TrueNAS and provision individual user directories on the client OS. 
 * Create a single SMB share and configure the ACL so that users can create individual directories on the share that inherit write access for the user and grant read access the administrator.
-* Create an SMB share using the **Private SMB datasets and shares** preset that can create per-user datasets under the umbrella of a single share when users access the share.
+* Create an SMB share using the **Private SMB datasets and shares** preset; and then create per-user datasets under the umbrella of a single share when users access the share.
 
 Creating an SMB home share requires configuring the system storage and provisioning users or joining Active Directory.
 
-### Adding Local Share Users
-Go to **Credentials > Users** and click **Add**.
-Create a new user name and password.
+## Adding Private SMB Datasets and Shares
+This option allows creating private share and datasets for the users that require the equivalent of the legacy home share.
+It is not intended for every user on the system.
+Setting up private SMB shares and datasets prevents the system from showing these to all users with access to the root level of the share.
+Examples of private SMB shares are those for backups, system configuration, and users or departments that need to keep information private from other users.
 
-By default, the user **Home Directory** title comes from the user account name and is added as a new subdirectory of **Home_Share_Dataset**.
+Before setting up SMB shares check system alerts to verify there are no errors related to connections to Active Directory.
```

Comment:
I wonder if we can qualify or deemphasize this somewhat, since it's only an issue for those using AD, not a general requirement for all users?

Reply:
I suspect this feature is in use by companies or enterprises rather than home users, and companies are likely using AD to provision users. I think we need to leave this as is until we receive feedback from users that this is an issue.